website/docs/admin_hosts_blender.md

```diff
@@ -0,0 +1,80 @@
+---
+id: admin_hosts_blender
+title: Blender
+sidebar_label: Blender
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Blender requirements
+Blender's integration requires to use **PySide2** module inside blender. Module is different for Blender versions and platforms so can't be bundled with OpenPype.
+
+### How to install
+
+:::info Permissions
+This step requires Admin persmission.
+:::
+
+<Tabs
+  groupId="platforms"
+  defaultValue="win"
+  values={[
+    {label: 'Windows', value: 'win'},
+    {label: 'Linux', value: 'linux'},
+  ]}>
+
+<TabItem value="win">
+
+Find python executable inside your Blender installation folder. Usually is located in **C:\\Program Files\\Blender Foundation\\Blender {version}\\{version}\\python\\bin\\python.exe** (This may differ in future blender version).
+
+Open Powershell or Command Prompt as Administrator and run commands below (Example commands are for **Blender 2.83** installed to default directory).
+```bash
+# Change directory to python executable directory.
+> cd C:\Program Files\Blender Foundation\Blender 2.83\2.83\python\bin
```

Comment:
This should clearer that the user should use the path they found in the first step